core/src/main/scala/org/apache/spark/storage/BlockManagerDecommissioner.scala

```diff
@@ -109,17 +109,21 @@ private[storage] class BlockManagerDecommissioner(
               s"to $peer ($retryCount / $maxReplicationFailuresForDecommission)")
             // Migrate the components of the blocks.
             try {
-              blocks.foreach { case (blockId, buffer) =>
-                logDebug(s"Migrating sub-block ${blockId}")
-                bm.blockTransferService.uploadBlockSync(
-                  peer.host,
-                  peer.port,
-                  peer.executorId,
-                  blockId,
-                  buffer,
-                  StorageLevel.DISK_ONLY,
-                  null) // class tag, we don't need for shuffle
-                logDebug(s"Migrated sub-block $blockId")
+              if (fallbackStorage.isDefined && peer == FallbackStorage.FALLBACK_BLOCK_MANAGER_ID) {
```

Comment:
is the fallback host name case insensitive?

Reply:
It is only set by this constant (e.g. as shared below), so keeping it case-sensitive should be fine.
https://github.com/apache/spark/blob/3022fd4ccfed676d4ba194afbfde2dd5ec1d348f/core/src/main/scala/org/apache/spark/storage/BlockManager.scala#L1684

There is also one area of precedence for comparing against this in a case-sensitive way:
https://github.com/apache/spark/blob/c2df895a8cbdfcbffc320e837ad1826933685912/core/src/main/scala/org/apache/spark/status/AppStatusStore.scala#L93